components/kyma-environment-broker/internal/dto.go

```diff
@@ -72,7 +73,28 @@ const (
 	Openstack       CloudProvider = "OpenStack"
 )
 
+type AutoScalerParameters struct {
+	AutoScalerMin  *int `json:"autoScalerMin"`
+	AutoScalerMax  *int `json:"autoScalerMax"`
+	MaxSurge       *int `json:"maxSurge"`
+	MaxUnavailable *int `json:"maxUnavailable"`
+}
+
+// FIXME: this is a makeshift check until the provisioner is capable of returning error messages
+// https://github.com/kyma-project/control-plane/issues/946
+func (p AutoScalerParameters) Validate() error {
+	if p.AutoScalerMin == nil || p.AutoScalerMax == nil {
+		return nil
+	}
+	if *p.AutoScalerMin > *p.AutoScalerMax {
+		return fmt.Errorf("Auto Scaler Max %v should be larger than Auto Scaler Min %v", *p.AutoScalerMax, *p.AutoScalerMin)
+	}
+	return nil
+}
+
 type ProvisioningParametersDTO struct {
+	AutoScalerParameters `json:",inline"`
```

Comment:
inlining and embedding `AutoScalerParams`, these are shared between provisioning and updating.